tests/ui/manual_is_ascii_check.fixed

```diff
@@ -15,6 +15,14 @@ fn main() {
     assert!('x'.is_ascii_alphabetic());
 
     assert!(matches!('x', 'A'..='Z' | 'a'..='z' | '_'));
+
+    assert!(&b'0'.is_ascii_digit());
```

Comment:
I dont think borrowing format is right, it should be `b'0'.is_ascii_digit()`, `&b'a'.is_ascii_lowercase()` etc, but i dont know how to do that.

Reply:
`.fixed` files are checked that they compile. If they don't the test would fail, so this wouldn't be a blocker. The ideal solution would be, to check if the expression is a reference [`ExprKind::AddrOf`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_hir/enum.ExprKind.html#variant.AddrOf), if so take the inner expression of `AddrOf` otherwise just snip it as it is :)